typescript/src/common/type.ts

```diff
@@ -0,0 +1,15 @@
+import { Dispatch } from "react";
```

Comment:
react 里的 Dispatch 定义是从 Hooks 引入之后增加的，以前的版本不存在
```suggestion
import { Dispatch } from "redux";
```